install/helm/agones/scripts/delete_agones_resources.sh

```diff
@@ -17,14 +17,11 @@
 namespaces=$@
 
 for ns in $namespaces; do
-
   # Building the list of pods we need to ensure are deleted.
   gs=$(kubectl -n $ns get gs -o jsonpath='{.items[*].metadata.name}')
 
   for g in $gs; do
     pod=$(kubectl -n $ns get po -l agones.dev/gameserver=$g -o jsonpath='{.items[*].metadata.name}')
-    # TODO(roberthbailey): Remove this after we cut Agones 0.12.0
-    pod=$(kubectl -n $ns get po -l stable.agones.dev/gameserver=$g -o jsonpath='{.items[*].metadata.name}')
```

Comment:
This variable hide the pod list retrieved by a previous line. This could be the reason why previously `make uninstall` timeout on gameservers crds which not able to be deleted.